include/circt/Dialect/RTL/Combinatorial.td

```diff
@@ -99,11 +99,15 @@ class UTVariadicRTLOp<string mnemonic, list<OpTrait> traits = []> :
 // Arithmetic and Logical Binary Operations.
 def AddOp : UTVariadicRTLOp<"add", [Commutative]>;
 def SubOp : UTBinRTLOp<"sub">;
-def MulOp : UTVariadicRTLOp<"mul", [Commutative]>;
-def DivOp : UTBinRTLOp<"div">;
-def ModOp : UTBinRTLOp<"mod">;
+def MulOp : UTVariadicRTLOp<"mulu", [Commutative]>;
+def MulSignedOp : UTVariadicRTLOp<"muls", [Commutative]>;
+def DivOp : UTBinRTLOp<"divu">;
+def DivSignedOp : UTBinRTLOp<"divs">;
```

Comment:
I'd recommend naming the MLIR Ops to align with the mneumonic.  DivUOp / DivSOp etc.